doc/user/content/sql/create-source/avro-kafka.md

```diff
@@ -19,10 +19,14 @@ topic.
 
 {{< diagram "create-source-avro-kafka.svg" >}}
 
-### format_spec
+### avro_format_spec
 
 {{< diagram "format-spec-avro-kafka.svg" >}}
 
+### format_spec
+
+{{< diagram "format-spec.svg" >}}
+
 ### key_constraint
```

Comment:
Does this page need both the avro_format_spec and the general format_spec?

Reply:
It's complicated, this is what it ends up looking this, with my highlights:

![image](https://user-images.githubusercontent.com/277161/126999343-1d7e95f3-db2c-4e1f-8060-73ed53754173.png)


I didn't want to take away from the highlighting of the fact that we are talking about Avro, but the Avro format spec is extremely verbose so unlike all the other Kafka sources I still wanted to pull it out into its own diagram. It was the closest I could think of to keeping the Avro format in line with other sources, e.g. CSV over Avro:

<img width="524" alt="image" src="https://user-images.githubusercontent.com/277161/126999691-b935929d-e349-4900-a6af-7aa374ce1736.png">

I'm happy to do something else, but readability of this page and consistency with other Avro sources are really fighting against each other in this one.